contributing/README.md

```diff
@@ -34,6 +34,15 @@ Start the title with `[ComponentName]` to identify which component a change affe
 [FooBar] Removes the deprecated fooWithBar:(Bar*)bar method.
 ```
 
+### Pull request continuous integration for new contributors
+
+Continuous integration will only be initiated automatically for the core team and
+recognized collaborators.
+
+All other pull requests must be labeled with `kokoro:force-run` by a member of the repo
```

Comment:
Rewording/suggestion: "by a member of (whatever membership is required)". I'm not sure what it means to be a member of the repository.

Reply:
I updated the language here. Please resolve accordingly once you've re-reviewed.

Reply:
What's the reason for excluding any future subteams of [mdc-ios](https://github.com/orgs/material-components/teams/mdc-ios) from automatic CI kick-offs?

Reply:
I'm not sure this content as-written precludes the possibility of us expanding the scope to include other teams in the future. As we stand right now, these are the two teams that define who has write access to the repo.

Reply:
Interesting. Then should we change the wording to just reflect "users with write access" instead of naming teams? That seems more future-proof.  Sorry for my initial suggestion, but I thought we were only granting kokoro privileges to the `core-ios-team` team.

Reply:
I've updated that wording accordingly below. The mechanism by which CI automatically runs is gated by our teams though so I'll keep that wording above.